### PR TITLE
Improve mqtt

### PR DESCRIPTION
**Pull request by wenwenchenbosch**
now mqtt must work, but subscribe via uuid part still missing, see todo https://github.com/boschresearch/kuksa.val/blob/35e04a73e8ce24fe02d6214b82915b86be1cd969/src/VssDatabase.cpp#L813

### CODE REVIEW

File: include/VssDatabase.hpp [hunk 1]
```diff
@@ -64,7 +64,7 @@ class VssDatabase : public IVssDatabase {
   void updateMetaData(WsChannel& channel, const std::string& path, const jsoncons::json& newTree) override;
   jsoncons::json getMetaData(const std::string &path) override;
 
-  void setSignalDBUS(const std::string &dbuspath, jsoncons::json value);
+  void setSignal(const std::string &dbuspath, jsoncons::json value);
```
Comment (SebastianSchildt) [inline include/VssDatabase.hpp]: why reverting this change

File: include/VssCommandProcessor.hpp [hunk 2]
```diff
@@ -43,6 +43,8 @@ class VssCommandProcessor : public IVssCommandProcessor {
   std::shared_ptr<SigningHandler> signer;
 #endif
 
+  std::string processSet(WsChannel& channel, const std::string& request_id, const std::string& path,
```
Comment (SebastianSchildt) [inline include/VssCommandProcessor.hpp]: why reintroducing deleted method?

File: include/VssDatabase.hpp (hunk 1 above)
Comment (wenwenchenbosch) [reply include/VssDatabase.hpp]: revert back

File: include/VssCommandProcessor.hpp (hunk 2 above)
Comment (wenwenchenbosch) [reply include/VssCommandProcessor.hpp]: removed

**Review by SebastianSchildt — APPROVED** [review]
Thank you

**Pull request merged**